modules/client/src/interfaces.ts

```diff
@@ -129,6 +123,14 @@ export interface IFreezePermissionDecodedParams
   permissionId: string;
 }
 
+export interface IHasPermissionParams {
+  daoAddressOrEns: string;
+  where: string;
+  who: string;
+  permission: string;
```

Comment:
is this the permission name? or its hex hash?
Adding `/** doc comments */` will help here